services/illuminance.md

```diff
@@ -0,0 +1,16 @@
+# Illuminance
+
+Measures light in Lux
+
+    identifier: 0x1e6ecaf2
+    extends: _sensor
+
+## Registers
+
+    ro light: u16.16 lux { typical_max=100000 } @ reading
```

Comment:
whoops; 100k doesn't fit in u16 (added a TODO to check this in spectool)

should we go u22.10 ?

Reply:
updated